src/settings/LostItemFeePolicy/components/EditSections/LostItemFeeSection/LostItemFeeSection.test.js

```diff
@@ -0,0 +1,474 @@
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.lostItemReturned,
+        expect.objectContaining({
+          htmlFor: 'Charge',
+          id: 'Charge-label',
+        }), {}
+      );
+    });
+
+    it('should render "replacement allowed" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.replacementAllowed,
+        expect.objectContaining({
+          htmlFor: 'replacementAllowed',
+          id: 'replacementAllowed-label',
+        }), {}
+      );
+    });
+
+    it('should render "replaced lost item processing fee" label', () => {
+      expect(Label).toHaveBeenNthCalledWith(
+        labelCallOrderByPlace.replacedLostItemProcessingFee,
+        expect.objectContaining({
+          htmlFor: 'replacedLostItemProcessingFee',
+          id: 'replacedLostItemProcessingFee-label',
+        }), {}
+      );
+    });
+
+    it('should render radio group', () => {
+      expect(RadioGroup).toHaveBeenCalled();
+    });
+
+    it('should render "chargeAmountItemPatron" text', () => {
+      expect(
+        within(screen.getByTestId('chargeAmountItemPatronColumn'))
+          .getByText(`${labelIds.chargeAmountItemPatron}?`)
+      ).toBeVisible();
+    });
+
+    it('should render "chargeAmountItemSystem" text', () => {
+      expect(
+        within(screen.getByTestId('chargeAmountItemSystemColumn'))
+          .getByText(`${labelIds.chargeAmountItemSystem}?`)
+      ).toBeVisible();
+    });
+
+    it('should render "returnedLostItemProcessing" text', () => {
+      expect(
+        within(screen.getByTestId('returnedLostItemProcessingFeeColumn'))
+          .getByText(labelIds.returnedLostItemProcessingFee)
+      ).toBeVisible();
+    });
+
+    it('should render "lostItemReturned" text', () => {
+      expect(
+        within(screen.getByTestId('lostItemReturnedColumn'))
+          .getByText(labelIds.lostItemReturned)
+      ).toBeVisible();
+    });
+
+    it('should render "replacementAllowed" text', () => {
+      expect(
+        within(screen.getByTestId('replacementAllowedColumn'))
+          .getByText(labelIds.replacementAllowed)
+      ).toBeVisible();
+    });
+
+    it('should render "replacedLostItemProcessingFee" text', () => {
+      expect(
+        within(screen.getByTestId('replacedLostItemProcessingFeeColumn'))
+          .getByText(labelIds.replacedLostItemProcessingFee)
+      ).toBeVisible();
+    });
+  });
+
+  describe('"edit lost item fee" accordion', () => {
+    [
+      false,
+      true,
+    ].forEach((lostItemFeeSectionOpen) => {
+      describe(`when "lostItemFeeSectionOpen" prop is ${lostItemFeeSectionOpen}`, () => {
+        it('should render accordion', () => {
+          render(
+            <LostItemFeeSection
+              {...defaultTestProps}
+              lostItemFeeSectionOpen={lostItemFeeSectionOpen}
+            />
+          );
+
+          expect(Accordion).toHaveBeenNthCalledWith(
```

Comment:
nit: use `toHaveBeenLastCalledWith` instead of `toHaveBeenNthCalledWith` and remove order map for `Accordion`.